skimage/util/process.py

```diff
@@ -0,0 +1,68 @@
+from multiprocessing import cpu_count
+
+import dask.array as da
+
+__all__ = ['process_chunks']
+
+
+def _get_chunks(shape, ncpu):
```

Comment:
@jni You are correct, I was thinking about this in one dimension. This currently gives you `ncpu * len(shape)` chunks. I'll fix this.

Also `_get_chunks` should return a tuple of tuples. One tuple for each dimension listing the size of each chunk in that dimension. So:

```python
>>> _get_chunks((4, 4), 4)
((2, 2), (2, 2))
>>> _get_chunks((4, 4), 4)
((2, 2), (4,))
>>> _get_chunks((5, 5), 2)
((2, 3), (5,))
>>> _get_chunks((2, 4), 2)
((1, 1), (4,))  # ? or it might be better to have ((2,), (2, 2))
```